Makefile

```diff
@@ -69,9 +69,9 @@ IS_PRIVATE ?= $(findstring private,$(_git_remote_urls))
 # Note that for everything except RC builds, VERSION will be set to the version
 # we'd use for a GA build. This is by design.
 ifneq ($(GIT_TAG_SANITIZED),)
-VERSION = $(shell printf "$(GIT_TAG_SANITIZED)" | sed -e 's/-.*//g')
+VERSION = $(shell printf "$(GIT_TAG_SANITIZED)" | sed -e 's/-.*//g' | sed -e 's/^[vV]//')
 else
```

Comment:
I'd have `$(patsubst v%,%, …)`.

Reply:
Actually, `VERSION = $(patsubst v%,%,$(firstword $(subst -, ,$(GIT_TAG_SANITIZED)))`

Reply:
I'm going to land this as-is and then make the change after 0.77.0.